### DISCUSSION TITLE
Placeholders on Location page

### DISCUSSION THREAD

**Discussion by meilensteindigital** [liedekef/events-made-easy#107] [General]
Hi there, somehow placeholders dont work on my Location page. What could be the reason for this?

https://avm-fobi.meilenstein-digital.de/locations/nuernberg/

Any help is appreciated.

Best
Thomas

**Comment by liedekef** [comment]
Where did you enter these placeholders? What is the value of "Default single location page format" (in the EME settings)?

**Reply by meilensteindigital** [reply]
I see, that i need to put the placeholders in “Default single location page format" in the EME settings ;) thank you for pointing me in the right direction

**Reply by liedekef** [reply]
You can put placeholders in the location description too, but then you'll need to activate the EME option to parse placeholders in the description (by default that does not happen)
